custom/icds_reports/migrations/0001_squashed_0110_aww_incentive_report_monthly.py

```diff
@@ -1,1887 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
why is this deleted?

Reply:
Sorry, @calellowitz I deleted it by mistake. I restored it and PR is ready to merge